### PR TITLE
PDG-36

### PR DESCRIPTION
**Pull request by Rhod3**

### CODE REVIEW

File: client/src/main/java/ch/heig/Constants.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  */
 public interface Constants {
     int SERVER_PORT = 8443;
-    String SERVER_URL = "https://localhost";
+    String SERVER_URL = "https://192.168.99.100";
```
Comment (Rhod3) [inline client/src/main/java/ch/heig/Constants.java]: Do we leave the standard docker-machine for Windows IP adress or do we stay with localhost ?

Comment (Farenjihn) [reply client/src/main/java/ch/heig/Constants.java]: We should hardcode an hostname (as if we deployed the service behind a reverse proxy) and either instruct the user to edit his hosts file if he wants to run the server locally or provide a way to change this constant dynamically with spring.

For an example of the second option, you can check out the language management that currently exists in the app.

Comment (Rhod3) [reply client/src/main/java/ch/heig/Constants.java]: I'll go with the first solution, as this would be the way to go if we would deploy our server.

**Comment by Farenjihn** [comment]
We can now change the server address by adding
```
--server-address={addr}
```
at the end of the command line to run the client.

**Pull request merged**